### ISSUE TITLE
Export issues: embedded images, bulleted + numbered lists

### ISSUE THREAD

**Issue by bjohnson747** [harvard-lil/h2o#390]
Headnote text has limited formatting support: bold works and italics works, but line breaks don't seem to work, nor bullet points. 

LEFT SIDE: export doc          / RIGHT SIDE: headnote in H2O

<img width="648" alt="screen shot 2018-04-19 at 2 36 33 pm" src="https://user-images.githubusercontent.com/12850389/41315925-e3e743d0-6e5e-11e8-8747-aa3d495630dd.png">

This limited formatting support works only on the top-level section's headnote that's exported:

In a full casebook export, the casebook's main headnote will show this partial formatting, but none of its sections or resource's headnotes support it. When a specific section is exported, that section's headnote has the partial formatting, but none of its subsections or resources do.
In other words, only the top-level section's headnote that's exported supports this partial formatting. 

Per Casey, this is the wiki for styling for the package we use https://github.com/karnov/htmltoword/wiki/Styles-and-classes

**Comment by abziegler** [comment]
@bjohnson747 please clarify: is this an issue with the export content or with the ability to format headnotes in-app?

**Comment by bjohnson747** [comment]
@abziegler export content - the headnote text editor supports these formatting options, per screenshot

**Comment by bjohnson747** [comment]
less relevant / clear than the newly created ticket #606 - closing.
